scripts/gen.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```

Comment:
IIRC, since you're running this script directly with `/bin/bash` in the `Makefile`, this line is unnecessary.

Instead, I think the Makefile should simply run `./scripts/gen.sh` instead of passing the script to `./bin/bash` explicitly, and let the script define which shell it uses.

Reply:
be07c7e